src/main/java/com/superdevs/HealthOMeter/controllers/ContactController.java

```diff
@@ -0,0 +1,45 @@
+package com.superdevs.HealthOMeter.controllers;
+
+import com.superdevs.HealthOMeter.dto.ContactDto;
+import com.superdevs.HealthOMeter.mappers.ContactMapper;
+import com.superdevs.HealthOMeter.service.ContactService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin("*")
+@RestController
+@RequestMapping("/hom")
+public class ContactController {
+
+    private static final Logger logger = LoggerFactory.getLogger(ContactController.class);
+
+    @Autowired
+    private ContactService contactService;
+
+    @Autowired
+    private ContactMapper contactMapper;
+
+    @GetMapping(value = "/contacts")
```

Comment:
OMG.. a po co wszystkie kontakty chcesz wyrzucać na front.. wyciek danych nam nie potrzebny ;)

Reply:
DTO will reaming as it is, because we don't know precisely what we need - it's better to have more in this context, than less, we need more data from UI or we need to meet and decide what we will use outside.
Regarding contacts:
it's strictly for testing purposes - to check if controller/mapper/dto is working, and it was easier to get all contacts than oneByOne when creating a new one. It will be only used in Postman, Front doesn't need it. It will be later deleted when not needed.